src/data/components.json

```diff
@@ -223,7 +226,8 @@
       "description": "Feature card is used to emphasize a piece of information with a card on the right-hand side and an image on the left-hand side.",
```

Comment:
Should there be a page for `feature-card`? It's not in the components list.

Reply:
The `feature-card`, `card-link`, and `card-in-card` all appear on the `Card` page as their own sections. Each should have a ResourceCard, but I'll double check to make sure